src/common/drydep.f90

```diff
@@ -79,7 +80,7 @@ subroutine drydep2(tstep, part)
 
   m = part%icomp
 !#### 30m = surface-layer (deposition-layer); sigma(hybrid)=0.996 ~ 30m
-  if (def_comp(m)%kdrydep == 1 .AND. part%z > 0.996) then
+  if (def_comp(m)%kdrydep == 1 .AND. part%z > surface_height_sigma) then
```

Comment:
Why don't you calculate a sigma_level_of_surface_30m alongside the sigma_level_of_surface_h and drop surface_height_sigma completely? (Or is that calculation too slow?)

Reply:
This is the original implementation and should be revised in the work on dry deposition